package.json

```diff
@@ -4,6 +4,7 @@
     "packages/*"
   ],
   "scripts": {
+    "publish-canary": "lerna publish --canary --npm-client=npm --registry=https://registry.npmjs.org/",
```

Comment:
This command publishes alpha versions of changed packages linked to specific commits, and without the autogenerated tags or changes to the package.json.

Useful for testing in Mavenlink.